autoload/limelight.vim

```diff
@@ -139,7 +139,7 @@ function! s:dim(coeff)
   let fg = synIDattr(synid, 'fg#')
   let bg = synIDattr(synid, 'bg#')
 
-  if has('gui_running') || exists('+termguicolors') || has('nvim') && $NVIM_TUI_ENABLE_TRUE_COLOR
+  if has('gui_running') || (exists('+termguicolors') && termguicolors)|| has('nvim') && $NVIM_TUI_ENABLE_TRUE_COLOR
```

Comment:
@junegunn 2 hours ago in gitter the neovim developers told me that `exists('+termguicolors')` is better if you want to mantain compatibility between nvim and vim. 

I'll fix the space.